changelogs/fragments/596-splunk-add-option-to-not-validate-cert.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - splunk - Add an option to allow not to validate certificate from HEC (https://github.com/ansible-collections/community.general/pull/596).
```

Comment:
```suggestion
  - splunk - add an option to allow not to validate certificate from HEC (https://github.com/ansible-collections/community.general/pull/596).
```

Reply:
Please write `slunk callback` instead of just `splunk` so it is clear that this is not about a module of that name. (We usually use the module name for modules, and the plugin name + type for plugins.)